src/usecase/client_user_ucase.go

```diff
@@ -98,10 +99,24 @@ func (cu *ClientUser) Add(c *gin.Context) { //nolint:dupl
 		return
 	}
 
+	// Checking for empty var, then check status and add user if status "deleted"
 	if existingUser.ID != uuid.Nil {
+		if userStatus == "deleted" {
+			user, _ := userRepo.Add(user)
+			_ = domain.ClientUser{
```

Comment:
ClientUser isn't used, look at the example above and decided if it's needed or not, don't leave unused variables like this